com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/ResolveClasspathsHandler.java

```diff
@@ -178,10 +181,10 @@ public void acceptSearchMatch(SearchMatch match) {
         for (int i = 0; i < unresolved.length; i++) {
             IRuntimeClasspathEntry entry = unresolved[i];
             if (entry.getClasspathProperty() == IRuntimeClasspathEntry.USER_CLASSES) {
-                IRuntimeClasspathEntry[] entries = JavaRuntime.resolveRuntimeClasspathEntry(entry, jproject);
+                IRuntimeClasspathEntry[] entries = JavaRuntime.resolveRuntimeClasspathEntry(entry, jproject, true);
                 for (int j = 0; j < entries.length; j++) {
 
-                    if (entries[j].getClasspathEntry().isTest()) {
+                    if (entries[j].getClasspathEntry().isTest() && !isRuntime(entries[j].getClasspathEntry())) {
```

Comment:
I believe you should always add runtime classes, regardless of their "test" attribute